README.md

```diff
@@ -14,20 +14,23 @@ Installation
 
 ### Distribution dependencies ###
 
-on Debian Stretch (9) :
+on Debian Stretch (9) with sudo:
 
+    sudo apt update
     sudo apt install git
-    sudo apt install python3 python-virtualenv python3-dev python-pip
+    sudo apt install python3 python3-venv python3-dev python3-pip
 
 ### Get the source ###
 
+get source from github:
+
     git clone https://github.com/tracim/tracim_backend.git
 
-### Setup Python Virtualenv ###
+go to *tracim_backend* directory:
 
-Go to *tracim* subdirectory:
+    pushd tracim_backend
```

Comment:
use `cd tracim_backend` ; i believe pushd is too "poweruser" for common users (even if it works, they will ask wtf is this tool?!)

Reply:
yes, you're right, i first thought about pushd because i need to go into a directory and them go back (in this case pushd make more sense) but it's not the case anymore since i replace virtualenv dir.